packages/components/src/Combobox/components/ComboboxContent/ComboboxContent.tsx

```diff
@@ -19,17 +20,33 @@ interface ComboboxContentProps {
    * actions
    */
   readonly children: ReactElement | ReactElement[];
+
+  /**
+   * pre selected option
+   * @default ""
+   * @optional
+   * @type string
+   * */
+  readonly preSelectedOption?: string;
```

Comment:
with the name, I think this makes perfect sense when we're using it for editing

what I'm wondering about is other ways you might want to use this prop. specifically I'm thinking about how you might clear a selection. I feel like that's a case we should support when I think of team assignment, or tags on a report - both of those it's perfectly valid to have nothing selected to either keep it unassigned or see your report with no tags reducing your results

maybe we don't want to use this prop to support that use case/flow which is totally fair but I think we _could_

I'm imagining the following
- click the trigger
- open the list
- choose something, setting the internal selection and doing whatever on the consumer side with that returned selection
- decide hey I don't want that anymore
- use a `setSelection` prop that will, as it suggests, set the internal selection - in this case setting it to `null`

to me that feels flexible where you could use it for editing and clearing? although what I've described is definitely more of a forwardedRef/imperativeHandle pattern where you'd do `myComboboxRef.current.setSelection(null)` or `myComboboxRef.current.setSelection(userThatWasAlreadySelected)`

now that I've typed it out though, do we even care about the internal state of this component? like imagine I have that form and I've used the value returned from `Combobox`. 

I can clear that myself... since `Combobox` has no visual representation of what's selected, what even would be the point of clearing the component's internal state? 🤔

Reply:
Hm I like this thought process. I  agree that the `setSelection` naming convention would be more flexible so I do think that's a good refactor. Although combobox technically wouldn't be linked to a visual representation of the output wouldn't we still want to clear the state of the combobox if we wanted to clear the visual output considering than one would be linked to the other?

Reply:
yeah... thinking on this a bit more, while we could get away with not clearing the internal state for single select things will definitely get awkward with multi select where there will be an intermediary visual representation of what is selected via the check boxes & count, even if it also has no final output

if we solely rely on the consumer to manage clearing/updating the data outside the component, you could end up in weird states where the piece of the form using `Combobox` might say "nothing selected" but then you open up the combobox and it still shows your previous selections